apple/DemoApp/Ferrostar Demo.xcodeproj/xcshareddata/xcschemes/iOS Demo.xcscheme

```diff
@@ -49,6 +49,13 @@
             ReferencedContainer = "container:Ferrostar Demo.xcodeproj">
          </BuildableReference>
       </BuildableProductRunnable>
+      <EnvironmentVariables>
```

Comment:
Feel free to drop this. It was actually relatively useful in the demo app, but obviously not a common need.

Reply:
Yeah. I was originally going to take it away, but figured it was actually a good default for debugging with the demo app 👍